src/builder.rs

```diff
@@ -173,6 +173,26 @@ impl<St: BuilderState> ConfigBuilder<St> {
             .insert(Expression::from_str(key.as_ref())?, value.into());
         Ok(self)
     }
+
+    ///Sets an override if value is Some(_)
```

Comment:
Missing whitespace after `///`.

Reply:
No worries. Comment formatting is important. My bad I did not notice!